include/onnxruntime/core/common/status.h

```diff
@@ -89,23 +89,23 @@ inline HRESULT StatusCodeToHRESULT(StatusCode status) noexcept {
     case StatusCode::INVALID_ARGUMENT:
         return E_INVALIDARG;
     case StatusCode::NO_SUCHFILE:
-        return __HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND);
+        return HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND);
     case StatusCode::NO_MODEL:
-        return __HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND);
+        return HRESULT_FROM_WIN32(ERROR_FILE_NOT_FOUND);
     case StatusCode::ENGINE_ERROR:
         return E_FAIL;
     case StatusCode::RUNTIME_EXCEPTION:
         return E_FAIL;
     case StatusCode::INVALID_PROTOBUF:
-        return __HRESULT_FROM_WIN32(ERROR_FILE_CORRUPT);
+        return HRESULT_FROM_WIN32(ERROR_FILE_CORRUPT);
     case StatusCode::MODEL_LOADED:
-        return __HRESULT_FROM_WIN32(ERROR_INTERNAL_ERROR);
+        return HRESULT_FROM_WIN32(ERROR_INTERNAL_ERROR);
     case StatusCode::NOT_IMPLEMENTED:
         return E_NOTIMPL;
     case StatusCode::INVALID_GRAPH:
-        return __HRESULT_FROM_WIN32(ERROR_FILE_CORRUPT);
+        return HRESULT_FROM_WIN32(ERROR_FILE_CORRUPT);
     case StatusCode::EP_FAIL:
-        return __HRESULT_FROM_WIN32(ERROR_INTERNAL_ERROR);
+        return HRESULT_FROM_WIN32(ERROR_INTERNAL_ERROR);
```

Comment:
`__HRESULT_FROM_WIN32` is a macro and `HRESULT_FROM_WIN32` is a constexpr function, seemed like an improvement